xml/System.Text.Json/JsonElement.xml

```diff
@@ -641,10 +723,13 @@
         <Parameter Name="value" Type="System.Single" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="value">Receives the value.</param>
+        <summary>Attempts to represent the current JSON number as a <see cref="T:System.Single" />.</summary>
+        <returns><see langword="true" /> if the number can be represented as a <see cref="T:System.Single" />, <see langword="false" /> otherwise.</returns>
+        <remarks>This method does not parse the contents of a JSON string value.
+On .NET Core this method does not return <see langword="false" /> for values larger than <see cref="F:System.Single.MaxValue" /> (or smaller than <see cref="F:System.Single.MinValue" />), instead <see langword="true" /> is returned and <see cref="F:System.Single.PositiveInfinity" /> (or <see cref="F:System.Single.NegativeInfinity" />) is emitted.</remarks>
```

Comment:
The remarks section should use the \<format> element and xrefs with ?displayProperty=nameWithType.

Reply:
```suggestion
On .NET Core this method does not return <xref:false?displayProperty=nameWithType> for values larger than <xref:System.Single.MaxValue?displayProperty=nameWithType> (or smaller than <xref:System.Single.MinValue?displayProperty=nameWithType>), instead <xref:true?displayProperty=nameWithType> is returned and <xref:System.Single.PositiveInfinity?displayProperty=nameWithType> (or <xref:System.Single.NegativeInfinity?displayProperty=nameWithType>) is emitted.
]]></format></remarks>
```

Reply:
@rpetrusha do I also use xref for "true" and "false"?

Reply:
It's preferable to use \<see langword…, @carlossanlop.